service/src/test/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/CopyGcsBucketDefinitionStepTest.java

```diff
@@ -121,22 +172,11 @@ public void testDoStep() throws InterruptedException {
     assertEquals(SOURCE_BUCKET_RESOURCE.getManagedBy(), destinationBucketResource.getManagedBy());
     assertEquals(
         SOURCE_BUCKET_RESOURCE.getApplicationId(), destinationBucketResource.getApplicationId());
-    assertEquals(
```

Comment:
why is this deleted?

Reply:
ah thanks for explaining. this's an example of why mocking is bad. :)  yeah let's assert the current pricate state is ACTIVE.